src/erl/lang/clojerl.Keyword.erl

```diff
@@ -85,7 +85,7 @@ apply(Keyword, Args) ->
 
 %% clojerl.IHash
 
-hash(Keyword) when is_atom(Keyword) ->
```

Comment:
This change is unrelated but useful. None of the other protocol functions are using a guard to check their argument is a an atom.